### PR TITLE
[Feature] addVolumeMount

### LINKED ISSUE #321: Adding mounts to redis containers
Hey,

I would like to add modules to the cluster, I can add the `loadmodule` directive as an `additionalRedisConfig` but it needs to point to the so file on the container and I could not find a way to add any additional ConfigMap mounts to the containers.

Any direction or suggestion will be much appreciated.
Thanks!

### PR DESCRIPTION
**Pull request by shubham-cmyk**
**Description**

Adding Volume mount to the pods. 

Fixes #321

### CODE REVIEW

**Comment by shubham-cmyk** [comment]
Signed the Commit

**Comment by shubham-cmyk** [comment]
I have tested the mouting 

Screenshot from 2022-11-23 17-47-22

The container are mounted with the volume

**Comment by shubham-cmyk** [comment]
@iamabhishek-dubey You can merge the PR

File: k8sutils/statefulset.go [hunk 1]
```diff
@@ -55,6 +56,8 @@ type containerParameters struct {
 	TLSConfig                    *redisv1beta1.TLSConfig
 	ReadinessProbe               *redisv1beta1.Probe
 	LivenessProbe                *redisv1beta1.Probe
+	AdditionalVolume             []corev1.Volume
+	UserMountPath                []corev1.VolumeMount
```
Comment (iamabhishek-dubey) [inline k8sutils/statefulset.go:60]: Let's call it `AdditionalMountPath`

**Review by iamabhishek-dubey — CHANGES_REQUESTED** [review]

**Comment by iamabhishek-dubey** [comment]
Also, it would be great if you can add an example for this in the [examples](https://github.com/OT-CONTAINER-KIT/redis-operator/tree/master/example) folder.

File: example/volume_Mount/redis-cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+          configMap:
+            name: example-configmap
+      mount:
+        - mountPath: /config
+          name: example-config
+  # nodeSelector:
+  #   kubernetes.io/hostname: minikube
+  # priorityClassName:
+  # Affinity:
+  # Tolerations: []
```
Comment (iamabhishek-dubey) [inline example/volume_Mount/redis-cluster.yaml:79]: Can we remove other unnecessary maps and objects, just like in the other examples folder? Also, let's keep the name of folder in `snake_case`.

Comment (iamabhishek-dubey) [reply example/volume_Mount/redis-cluster.yaml:79]: Also, add one example for standalone as well.

**Pull request closed without merging**